src/renderer/components/playlist-info/playlist-info.js

```diff
@@ -93,11 +93,11 @@ export default Vue.extend({
     this.infoSource = this.data.infoSource
 
     // Causes errors if not put inside of a check
-    if (typeof (this.data.viewCount) !== 'undefined') {
+    if (typeof (this.data.viewCount) !== 'undefined' && !isNaN(this.data.viewCount)) {
```

Comment:
I think you can remove the undefined check here as isNaN will return true if the value is undefined

Reply:
![image](https://user-images.githubusercontent.com/1018543/206965983-3f6d8f5f-55b1-4bc7-a038-e432b0ba6b21.png)